platform_plugin_ontask/conf/locale/es_419/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,98 @@
+# edX translation file.
+# Copyright (C) 2024 EdX
+# This file is distributed under the GNU AFFERO GENERAL PUBLIC LICENSE.
+# EdX Team <info@edx.org>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: 0.1a\n"
+"Report-Msgid-Bugs-To: openedx-translation@googlegroups.com\n"
+"POT-Creation-Date: 2023-06-13 08:00+0000\n"
+"PO-Revision-Date: 2023-06-13 09:00+0000\n"
+"Last-Translator: \n"
+"Language-Team: openedx-translation <openedx-translation@googlegroups.com>\n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: exceptions.py:9
+msgid "The course key is not valid."
+msgstr "El course key no es válido."
+
+#: exceptions.py:23
+msgid "The course does not exist."
+msgstr "El curso no existe."
+
+#: exceptions.py:40
+msgid ""
+"The OnTask API Auth Token is not set for this course. Please set it in the "
+"Advanced Settings of the course."
+msgstr ""
+"El token de autenticación de la API de OnTask no está configurado para este "
+"curso. Por favor, configúrelo en la Configuración Avanzada del curso."
+
+#: exceptions.py:60
+msgid ""
+"The OnTask Workflow ID is not set for this course. Please set it in the "
+"Advanced Settings of the course."
+msgstr ""
+"El ID del workflow de OnTask no está configurado para este curso. "
+"Por favor, configúrelo en la Configuración Avanzada del curso."
+
+#: static/html/ontask.html:7
+msgid ""
+"Load the data from the course into OnTask. This will create a new table in "
+"OnTask with the data summary configured in the plugin settings. The default "
```

Comment:
I thought we removed the "in the plugin settings" part

Reply:
I thought so too hahaha, I corrected it: https://github.com/eduNEXT/platform-plugin-ontask/commit/2a796aa78e18102e28c2858ea868f63fbe60e7cf